meta-facebook/yv35-hd/src/platform/plat_hook.c

```diff
@@ -129,3 +131,25 @@ bool post_adm1278_pwr_read(uint8_t sensor_num, void *args, int *reading)
 
 	return true;
 }
+
+bool post_ltc4282_cur_read(uint8_t sensor_num, void *args, int *reading)
+{
+	sensor_val *sval = (sensor_val *)reading;
+	float val = sval->integer + (sval->fraction * 0.001);
+	val = ADJUST_LTC4282_CURRENT(val);
+	sval->integer = (int16_t)val;
+	sval->fraction = (val - sval->integer) * 1000;
+
+	return true;
+}
+
+bool post_ltc4282_pwr_read(uint8_t sensor_num, void *args, int *reading)
```

Comment:
Check reading to see if it's NULL.

There's a common macro we've added for that which also logs errors.

Reply:
OK, will use the common macro to check if "reading" is NULL.